components/molecules/AlgoliaResults/Refinements/CustomClearRefinements.js

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types'
+import {connectCurrentRefinements} from 'react-instantsearch-dom'
+import styles from '../AlgoliaResults.module.css'
+
+/**
+ * Custom display of Algolia [ClearRefinements](https://www.algolia.com/doc/api-reference/widgets/clear-refinements/react/) widget.
+ */
+function ClearRefinements({items, refine}) {
+  return (
+    <>
+      {!!items.length && (
```

Comment:
@dcooney I think this needs the optional chaining operator? Doesn't this say, "bang bang you're a boolean do you have length?"

```suggestion
      {!!items?.length && (
```

Reply:
Seems to work either way. Your comment makes sense though.